network/templates/network/partials/instrument_unknown_statuses.html

```diff
@@ -0,0 +1,3 @@
+<div class="alert alert-warning">
+    There are {{ num_unknown_statuses }} observations with unknown status.
+  </div>
```

Comment:
This html was copied from the tom_targets app.

Reply:
Sorry, I should have been more specific here.
What you had was perfectly fine, but it's considered "proper" styling to include an empty line at the end of a code file. 
This will often be flagged in compilers and IDEs as bad formatting. 

I think there are a number of legacy reasons for this preference, but all I was looking for was a new empty line at the end of the file.